package.json

```diff
@@ -26,7 +26,7 @@
     "lodash-es": "^4.17.21",
     "nerf-dart": "^1.0.0",
     "normalize-url": "^8.0.0",
-    "npm": "^10.5.0",
+    "npm": "^11.0.0",
```

Comment:
Possibly this should be bumped to `^11.4.2` for clarity, due to https://github.com/semantic-release/npm/issues/966#issuecomment-2987561315, although `package-lock.json` does lock npm to `11.4.2` in the PR.